stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -242,6 +244,18 @@ public static void addMultipartsToCarbonMessage(HTTPCarbonMessage httpRequestMsg
         return abstractNativeFunction.getBValues(entity);
     }
 
+    /**
+     * Create file struct and set it to the entity.
+     *
+     * @param context Represent ballerina context
+     * @param entity  Represent an entity
+     */
+    private static void addFileHandlerToEntity(Context context, BStruct entity) {
```

Comment:
`FileHandler` could give the wrong impression here. This simply sets a `File` struct as the `Entity` body right?

Reply:
Addressed with b98823c.